app/app.go

```diff
@@ -132,6 +132,9 @@ import (
 	"github.com/sedaprotocol/seda-chain/app/keepers"
 	appparams "github.com/sedaprotocol/seda-chain/app/params"
 	"github.com/sedaprotocol/seda-chain/docs"
+	dataproxy "github.com/sedaprotocol/seda-chain/x/data-proxy"
+	dataproxyKeeper "github.com/sedaprotocol/seda-chain/x/data-proxy/keeper"
+	dataproxyTypes "github.com/sedaprotocol/seda-chain/x/data-proxy/types"
```

Comment:
Can't find the doc but I think the convention is to keep package import names in lower case. So `dataproxykeeper` and `dataproxytypes`